### PR TITLE
Add initial classes

### PR DESCRIPTION
**Pull request by Cerdtruir**
project from: oop_school_library/1_implement_the_classes.md

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    return true if of_age? || @parent_permission
+
+    false
```
Comment (ikraamg) [inline person.rb:15]: Test out if you need the `false` here and below. 

By default. I think `nil` is returned at the end of the function, which is falsy.

File: person.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Person class
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
```
Comment (ikraamg) [inline person.rb]: Unless needed, rather use keyword arguments for all the arguments as it makes the code read better.

**Review by ikraamg — APPROVED** [review]
Nice!

File: person.rb (hunk 1 above)
Comment (Cerdtruir) [reply person.rb:15]: Yeah, i read its better practice to use false when using booleans not passing to something else

File: person.rb (hunk 2 above)
Comment (Cerdtruir) [reply person.rb]: ok cool, will do this

File: person.rb (hunk 1 above)
Comment (ikraamg) [reply person.rb:15]: Makes sense 👍

**Pull request merged**